tests/test_bundle.py

```diff
@@ -419,6 +419,20 @@ def _test_bundle_put(self, replica: Replica, fixtures_bucket: str):
                 bundle_version,
             )
 
+        with self.subTest(f'{replica}:  manifest response testing'):
```

Comment:
can you move this subtest after the subtest in line 436

Reply:
Complete, awaiting test results.